taipy/gui/renderers/user_control.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022 Avaiga Private Limited
```

Comment:
I suspect we need a specific package for these.
taipy.gui.extension?

Then taipy.gui.extension.Element and ElementLibrary